<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextAnnotationAttributesWidget</class>
 <widget class="QWidget" name="TextAnnotationAttributesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Annotation Attributes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="4">
       <widget class="QFontComboBox" name="pFont">
        <property name="toolTip">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="pRightAlign">
        <property name="toolTip">
         <string>Align Right</string>
        </property>
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="pLeftAlign">
        <property name="toolTip">
         <string>Align Left</string>
        </property>
        <property name="text">
         <string>Left</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Alignment</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QToolButton" name="pItalic">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="toolTip">
         <string>Italic</string>
        </property>
        <property name="text">
         <string>I</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="pCenterAlign">
        <property name="toolTip">
         <string>Align Center</string>
        </property>
        <property name="text">
         <string>Center</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QToolButton" name="pColor">
        <property name="toolTip">
         <string>Pick Text Color</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>..</string>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QFrame" name="pSelectedColor">
        <property name="toolTip">
         <string>Text Color</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="pSize">
        <property name="toolTip">
         <string>Font Size</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="pBold">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Bold</string>
        </property>
        <property name="text">
         <string>B</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
